examples/check_published.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'd add a short comment at the top of the file for future contributors explaining what this does.